### PR TITLE
roll out Profile-V3

### PR DESCRIPTION
**Pull request by daiagi**
**Thank you for your contribution** to the [Koda - Generative Art Marketplace](https://kodadot.xyz).

👇 __ Let's make a quick check before the contribution.

## PR Type

- [x] Feature

## Needs QA check

- @kodadot/qa-guild please review

## Context

- [x] Ref #9876
- [x] After #10169

#### Did your issue had any of the "$" label on it?

- [x] Fill up your DOT address: [Payout](https://canary.kodadot.xyz/dot/transfer/?target=1cAsKZYNRb8dkSCpn4eVkEn6ycNZTGoDo5dGDgB8J1UUWK8)

### CODE REVIEW

**Review by Jarsen136 — APPROVED** [review]

**Comment by prury** [comment]
only this for now: 
capital C
image

**Comment by prury** [comment]
maybe remove the buynow=false at the end: 
image

**Comment by prury** [comment]
other than the above, works for me, i have added some small visual bugs on visual PR

**Comment by daiagi** [comment]
>other than the above, works for me, i have added some small visual bugs on visual PR

Fixed.
Thank you :)

File: components/profile/ProfileActivitySummery.vue [hunk 1]
```diff
@@ -123,65 +101,14 @@ useLazyAsyncData('stats', async () => {
     return
   }
 
+  const listEvents: number[] = data.value.listed
```
Comment (preschian) [inline components/profile/ProfileActivitySummery.vue:104]: add types to `listed` and `obtained.totalCount`

**Review by preschian — APPROVED** [review]

File: components/profile/ProfileActivitySummery.vue (hunk 1 above)
Comment (daiagi) [reply components/profile/ProfileActivitySummery.vue]: Hey @preschian 

great suggestion

following this suggestion, I think it's a good idea to typify the various graphql queries we have

i have started in this PR

and will create a follow up issue

**Comment by daiagi** [comment]
@prury 

merge conflicts are already resolved

File: components/profile/ProfileActivitySummery.vue (hunk 1 above)
Comment (daiagi) [reply components/profile/ProfileActivitySummery.vue]: resolving conversation...

Comment (preschian) [reply components/profile/ProfileActivitySummery.vue]: no worries, I also create similar issue https://github.com/kodadot/nft-gallery/issues/10185

**Pull request merged**